docs/en/observability/apm/troubleshooting/common-problems.asciidoc

```diff
@@ -282,10 +283,42 @@ If the service map is not showing an expected connection between the client and
 it's likely because you haven't configured
 {apm-rum-ref}/distributed-tracing-guide.html[`distributedTracingOrigins`].
 
-
 This setting is necessary, for example, for cross-origin requests.
 If you have a basic web application that provides data via an API on `localhost:4000`,
 and serves HTML from `localhost:4001`, you'd need to set `distributedTracingOrigins: ['https://localhost:4000']`
 to ensure the origin is monitored as a part of distributed tracing.
 In other words, `distributedTracingOrigins` is consulted prior to the APM agent adding the
 distributed tracing `traceparent` header to each request.
+
+// This info is not confirmed, but hopefully provides a structure
+// for product/dev to react to and follow!
```

Comment:
@MiriamAparicio Can you confirm that this is all correct too?

Reply:
hi, yes this seems correct

Reply:
Thanks!